src/component/contents/DraftEditorBlocks.react.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
```

Comment:
I'm not sure to see how you want to implement this without rendering the inner blocks of a block.

What do you suggest?